### DISCUSSION TITLE
_variables.scss / $site-width

### DISCUSSION THREAD

**Discussion by a0damasco** [Lednerb/bilberry-hugo-theme#363] [Q&A]
Sorry if the question is very silly... I have no experience in this.
I modify _variables.scss: `$site-width: 800px` to `$site-width: 1800px` and nothing changes. I tried in site folder\themes\bilberry-hugo-theme\assets\sass\_variables.scss and also using a copy in site folder\assets\sass\_variables.scss
I'm sure I'm doing something wrong, but I can't figure out what.
Thanks in advance!

**Comment by darrenaddy** [comment]
Someone correct me, if I'm wrong.
I'm new to SASS also. I believe you will not see the change until you BUILD
your static site... unless:
What are you using for your IDE? If you use the free Visual Studio Code
there is an extension called Live Sass Compiler that will show your changes
to you LIVE.
(Note: there are two identically named VS Code extensions. The one by
Ritwick Dey is the original but is reportedly no longer being updated. The
one by Glenn Marks is being updated and has 5 stars.)

Also, not sure if you typo'd or what, but the scss partials must be named:
 _variables.scss
and the path must be identical to the one in the theme to overwrite it (by
precedence).
So the THEME path is: assets/sass/_variables.scss
so you would need to overwrite with your
rootSite/assets/sass/_variables.scss

Personally, I would copy the ENTIRE sass folder of files to rootSite/assets
even if you are changing only one file (for now).

Hope this helps.

**Comment by igor-baiborodine** [comment]
@a0damasco, please check the [Customs Colors and Fonts](https://github.com/Lednerb/bilberry-hugo-theme#custom-colors-and-fonts) section in README.
